### DISCUSSION TITLE
Feature Reqeust Merged Vesrions / duplicates of Episodes

### DISCUSSION THREAD

**Discussion by Doenergott** [CyferShepard/Jellystat#31] [General]
First of all i want to say thanks for this Project. The actual Version looks very promising.

For my Jellyfin Libraries i'm using Merged Versions for multiple versions of one Episode/Season/Movie.
It would be nice to implement duplicates of episodes and show them also in the File Path of the Episodes.

If i start a sync i only get Errors in the log:
Error performing bulk insert:Error: error: duplicate key value violates unique constraint "jf_library_seasons_pkey"
Error performing bulk insert:Error: error: duplicate key value violates unique constraint "jf_library_episodes_pkey"

The Episode in Jellystat shows only the properties of one of the files.

**Comment by CyferShepard** [comment]
Hi @Doenergott , thank you so much. 

I just want to get a bit more info so i can try and replicate this on my side for testing. When you say Merged Versions, are you using a plugin that merges multiple versions of the same file/folder? Some examples would be great on how i can create a similar test case on my side.

**Comment by Doenergott** [comment]
Sure,
yes i am using this plugin in Jellyfin: https://github.com/danieladov/jellyfin-plugin-mergeversions

Inside of my library for tv-shows i added two directorys like:
/data/tv & /data/tv4k

For every Episode i am storing one 1080p version and one 4k Version.

**Comment by CyferShepard** [comment]
Thanks, il replicate this setup on my side and work on a patch
